docker-app/qfieldcloud/core/cron.py

```diff
@@ -76,3 +77,27 @@ def do(self):
             },
             output="Job unexpectedly terminated.",
         )
+
+
+class DeleteDanglingProjectPackagesJob(CronJobBase):
```

Comment:
Just to be clear, by "dangling" you mean a package where a newer package exists ? (in my understanding, "dangling" would rather mean a package where the project doesn't even exist anymore)

If so, maybe call this "obsolete" instead ?

Reply:
Agree that dangling might be misinterpreted and "obsolete" is better.